pkg/services/sqlstore/annotation_test.go

```diff
@@ -14,275 +15,241 @@ func TestAnnotations(t *testing.T) {
 	mockTimeNow()
 	defer resetTimeNow()
 	InitTestDB(t)
+	repo := SqlAnnotationRepo{}
+
+	t.Run("Testing annotation create, read, update and delete", func(t *testing.T) {
+		t.Cleanup(func() {
+			_, err := x.Exec("DELETE FROM annotation WHERE 1=1")
+			require.NoError(t, err)
+			_, err = x.Exec("DELETE FROM annotation_tag WHERE 1=1")
+			require.NoError(t, err)
```

Comment:
I would actually use `assert.NoError` in the cleanup function, since if one cleanup action fails, you want to continue with the remaining ones.
```suggestion
			assert.NoError(t, err)
			_, err = x.Exec("DELETE FROM annotation_tag WHERE 1=1")
			assert.NoError(t, err)
```

Reply:
👍  Fixed.